### PR TITLE
chore: Switch to Standard Ruby

### PR DESCRIPTION
**Pull request by jonallured**
This PR migrates Rosalind to use Standard Ruby rather than RuboCop and follows our patterns for making this change.

Along the way I did make some changes to how the default rake task works but ultimately it's the same: it runs all linters/tests. @anandaroop lmk if you see any issues with this!

/cc @artsy/amber-devs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -132,7 +132,7 @@
       "git add"
     ],
     "**/*.rb": [
-      "bundle exec rubocop -a",
+      "bundle exec standard -a",
```
Comment (lidimayra) [inline package.json:135]: Shouldn't it be `bundle exec standardrb` (or `bundle exec rake standard`)?

Comment (jonallured) [reply package.json]: Good catch, thanks @lidimayra!! 🙏

**Review by lidimayra — APPROVED** [review]
LGTM!! 🎉

**Review by anandaroop — APPROVED** [review]
Nice!

**Pull request merged**